CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ fantastic project. There are several ways to contribute:
 * write more unittests (we dare you to do that :godmode:)
 
 Before attempting any development, please install HADDOCK3 following the
-instructions in the [INSTALL](INSTALL.md) file. Afterwards, follow the
+instructions in the [INSTALL](https://github.com/haddocking/haddock3/blob/main/docs/INSTALL.md) file. Afterwards, follow the
```

Comment:
Here, there is no need to use the full url, as the root is always `https://github.com/haddocking/haddock3/blob/main/`

Could be simplified to:
`[INSTALL](docs/INSTALL.md)`

Reply:
Ha, thanks! Changed it just now